clients/lcdproc/main.c

```diff
@@ -248,6 +250,9 @@ main(int argc, char **argv)
 			case 'f':
 				foreground = TRUE;
 				break;
+			case 'k':
+				showhostname = FALSE;
+				break;
```

Comment:
If you add a command line option, it would need to be documented in the man page.

However unless there is a specific reason why you need a command line option, I recommend to only control this setting from the configuration file.

Reply:
Ok :) I will remove it